pkg/daemon/utils/network.go

```diff
@@ -321,6 +321,28 @@ func EnableIPForward(family int) error {
 	if family == netlink.FAMILY_V4 {
 		return ip.EnableIP4Forward()
 	}
+
+	v6DefaultRoute, err := GetDefaultRoute(netlink.FAMILY_V6)
```

Comment:
how about checking and modifying accept_ra in `EnableIP6Forward?`

Reply:
`EnableIP6Forward` is in a third-party module, which cannot be modified.